mujoco_warp/_src/io.py

```diff
@@ -296,6 +297,9 @@ def put_model(mjm: mujoco.MjModel) -> types.Model:
 
     nxn_pairid[pairid] = i
 
+  convex_collision_pair = _CONVEX_COLLISION_FUNC & set(zip(mjm.geom_type[geom1], mjm.geom_type[geom2]))
```

Comment:
is there a way to just store a tuple of the distinct geom types here so we don't have to import `collision_convex`?  that way perhaps we can reuse this same mechanism for collision_sdf and collision_primitive?